src/Command/Project/ProjectCreateCommand.php

```diff
@@ -216,29 +258,57 @@ protected function getAvailableRegions($runtime = false)
         return $regions;
     }
 
+    /**
+     * Return the catalog.
+     *
+     * The default list is in the config `service.catalog`. This is
+     * replaced at runtime by an API call.
+     *
+     * @param bool $runtime
+     *
+     * @return array
+     */
+    protected function getAvailableCatalog($runtime = false)
+    {
+        if ($runtime) {
+            $catalog = [];
+            foreach ($this->api()->getClient()->getCatalog()->getData() as $item) {
+                if ($item['info'] && $item['template']) {
+                    $catalog[$item['template']] = $item['info']['name'];
+                }
+            }
+            $catalog['empty'] = 'Empty Project';
+        } else {
+            $catalog = (array) $this->config()->get('service.catalog');
+        }
+
+        return $catalog;
+    }
+
     /**
      * Returns a list of ConsoleForm form fields for this command.
      *
      * @return Field[]
      */
     protected function getFields()
     {
-        return [
-          'title' => new Field('Project title', [
+        $fields = [];
```

Comment:
these format changes to `getFields()` are unnecessary

Reply:
reverted